.github/CODEOWNERS

```diff
@@ -51,3 +51,10 @@
 /airbyte-integrations/connectors/destination-s3/ @airbytehq/destinations
 /airbyte-integrations/connectors/destination-snowflake/ @airbytehq/destinations
 /airbyte-integrations/connectors/destination-redshift/ @airbytehq/destinations
+
+# Python critical connectors
+/airbyte-integrations/connectors/source-facebook-marketing/ @airbytehq/critical-connectors
```

Comment:
I have this error on my side. I'm would like to update the team visibility following [this guide](https://docs.github.com/en/organizations/organizing-members-into-teams/changing-team-visibility) but I can't select teams at step 4.

![image](https://github.com/airbytehq/airbyte/assets/3360483/27a49382-b08c-4fd6-a9a5-a58851964d44)